src/daq_config_server/app.py

```diff
@@ -142,4 +162,14 @@ def health_check():
 
 
 def main():
+    # Set up logging with defaults, or using config.yaml if it exists
+    if os.path.isfile(CONFIG_PATH):
+        with open(CONFIG_PATH) as f:
+            data = yaml.safe_load(f)
+            logging_config = LoggingConfig(**data)
+    else:
+        logging_config = LoggingConfig()
+
+    set_up_logging(logging_config)
+
     uvicorn.run(app="daq_config_server.app:app", host="0.0.0.0", port=8555)
```

Comment:
e.g. for the config change (which can be a later change), you might need to change this when doing dev work locally if you need something else on this port, so this should be configurable. Or you want to limit requests to just localhost loopback traffic